src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -61,16 +57,14 @@ function CustomizedInputBase(props) {
         <FormControl className={classes.formControl}>
           <NativeSelect
             disableUnderline
-            value={state.filter}
-            onChange={handleChange('filter')}
+            value={selectedFilter}
+            onChange={(event) => onFilterChange(event)}
             name="filter"
             className={classes.selectEmpty}
             inputProps={{ 'aria-label': 'filter' }}
           >
-            <option value="">None</option>
             <option value="author">Author</option>
-            <option value="filter2">Filter2</option>
-            <option value="filter3">Filter3</option>
+            <option value="post">Post</option>
```

Comment:
Let's make this `selected`.

Reply:
Is this to set the default value? If it is, I tried it, but it was giving me issues. I instead set the initialState in `SearchPage.js` using the following line `const [filter, setFilter] = useState('author');`

Reply:
Yeah, for the default. Whatever works for making it a default.

Reply:
Got it, default to posts